### ISSUE TITLE
Extension generating npm-debug.log file at the root of project

### ISSUE THREAD

**Issue by nishant-labs** [editorconfig/editorconfig-vscode#200]
It is so annoying that editorconfig extension is generating multiple random npm-debug.log.<random number> file at the root of project if .editorconfig file is not present at project root.

Example:- npm-debug.log.4128042918

FYI -  I don't have any global config as well.

## Issue

|             | Visual Studio Code | editorconfig-vscode |
|-------------|--------------------|---------------------|
| **Version** | `1.26.1 (1.26.1)`            | `0.12.4`             |

### Root `.editorconfig` File
_Not present_

Are there any other relevant `.editorconfig` files in your project? _No_

| Visual Studio Code Setting     | Default | User    | Workspace |
|--------------------------------|---------|---------|-----------|
| `editor.insertSpaces`          | `true`  | `____`  | `____`    |
| `editor.tabSize`               | `4`     | `_`     | `_`       |
| `editor.trimAutoWhitespace`    | `true`  | `____`  | `____`    |
| `files.autoSave`               | `"off"` | `"___"` | `"___"`   |
| `files.insertFinalNewline`     | `false` | `_____` | `_____`   |
| `files.trimTrailingWhitespace` | `false` | `_____` | `_____`   |

### File opened

_When any file open_

### Expected behavior

_If no config file is present editorconfig-vscode extension should be like switched off_

### Actual behavior

_multiple npm-debug.log.<random number> is created, if I leave after some time root folder is full of npm-debug.log files_

### Additional comments or steps to reproduce

Just open any project without .editorconfig file :)

Thanks
Nishant

**Comment by jedmao** [comment]
1. I've never had this problem on projects w/o an `.editorconfig` file.
1. If no config file is present, this extension should absolutely _NOT_ "be like switched off," because `.editorconfig` files could very well exist outside/above the project root.
1. Can you disable any/all other extensions before labeling this extension as the culprit?
1. What are the contents of `npm-debug.log`?
1. Try opening the output tab in vscode and selecting "EditorConfig". Do you see any valuable messaging there?

**Comment by nishant-labs** [comment]
@jedmao I was about to comment that it may not be the issue with editorconfig extension.

The story is it started happening after installation of editorconfig and as well as vs code upgrade, so assumed its because of it. And when I removed this plugin (and reloaded vs code) yesterday,  npm-debug.log creation stopped. However, today when I restarted vs code, it happened again, I am not sure if it is with vs code or some plugin.

npm-debug.log is completely empty, this is what annoys me so much.

My apologies for a wrong bug report, I will try to create one with vs code repo itself for their analysis.

Thanks,
Nishant
